packages/kubernetes-1.29/prestart-pull-pause-ctr.conf

```diff
@@ -6,4 +6,5 @@ ExecStartPre=/usr/bin/host-ctr \
     pull-image \
     --source=${POD_INFRA_CONTAINER_IMAGE} \
     --registry-config=/etc/host-containers/host-ctr.toml \
-    --skip-if-image-exists=true
+    --skip-if-image-exists=true \
+    --labels="io.cri-containerd.pinned=pinned"
```

Comment:
Unless it's going to cause some other issue, we should set this for all k8s versions, not just 1.29.

Reply:
Yes, this can be set for all versions